backend/services/team.py

```diff
@@ -161,6 +188,20 @@ def get_team_with_credentials(self, name: str, password: str) -> Team:
             raise InvalidCredentialsException("Incorrect credentials. Please try again")
         return team
 
+    def delete_all_teams(self):
+        """Deletes all teams"""
+        self._session.exec(delete(Team))
```

Comment:
Deleting the whole team table may be what was causing those commands to fail after running the `delete_all_teams` script. Maybe instead you could delete all rows from the table (but not the table itself)